app/views/google_analytics.scala.html

```diff
@@ -0,0 +1,27 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@(sessionId: String, langCode: String, url: String)
+
+        ga('set', 'dimension5', '@langCode');
+        ga(function(tracker) {
+        var clientId = tracker.get('clientId');
+        ga('set', 'dimension66', clientId);
+        });
+        ga('set', 'page', '@url');
+        ga('set', 'anonymizeIp', true);
```

Comment:
You can remove this as it's in the main template

Reply:
done

Reply:
done